### PR TITLE
Add RGB and HSL web support

### PR DESCRIPTION
**Pull request by samuelselleck**
Added RGB and HSL text support on the web. Since color representation on the web is consistent it might be worth doing the translations for fill/text/links etc in the same place at some point, but I don't feel familiar enough with the code yet to dare try to do that :)

### CODE REVIEW

**Comment by samuelselleck** [comment]
Nevermind this needs to be done in the non-transpiled version. Will fix tomorrow!

File: pax-chassis-web/interface/public/pax-chassis-web-interface.js [hunk 1]
```diff
@@ -704,10 +704,20 @@ var Pax = (() => {
...
             newValue = `hsla(${p[0] * 255},${p[1] * 255},${p[2] * 255},${p[3] * 255})`;
+          } else if (style.fill.Rgb != null) {
+            let p = style.fill.Rgb;
+            newValue = `rgb(${p[0] * 255},${p[1] * 255},${p[2] * 255})`;
+          } else if (style.fill.Hsl != null) {
+            let p = style.fill.Hsl;
+            newValue = `hsl(${p[0] * 255},${p[1] * 255},${p[2] * 255})`;
+          } else {
+            console.log("unsupported color type!");
+            newValue = `rgb(255, 0, 255)`;
```
Comment (warfaj) [inline pax-chassis-web/interface/public/pax-chassis-web-interface.js:718]: Let's throw an error here instead.
`throw new Error("Unsupported Color Format");`

**Review by warfaj — APPROVED** [review]
Small nit but otherwise looks good! Great job

**Comment by samuelselleck** [comment]
Done :)

**Pull request merged**